node/serverless.yml

```diff
@@ -0,0 +1,9 @@
+service: examples
+provider:
+  name: aws
+  runtime: nodejs6.10
+  region: us-west-2
+  timeout: 30
+functions:
+  express:
+    handler: express.handler
```

Comment:
if including a serverless config for the express function, shouldn't we show how you set the IOPIPE_TOKEN env var?